isisd/isis_tlvs.c

```diff
@@ -4366,6 +5153,75 @@ static int pack_tlv_router_cap(const struct isis_router_cap *router_cap,
 	}
 #endif /* ifndef FABRICD */
```

Comment:
not specific to this code ... I'd really like to take fabricd out and add dist-opt-flood :-)